### PR TITLE
Feature/880 native for Number format

### PR DESCRIPTION
**Pull request by mahmed0715**
Please check the native part, I tried to fix all the places I see, if you find anything please report.
@norbertschuler

### CODE REVIEW

File: app/components/PlantProjects/PlantProjectSnippet.native.js [hunk 1]
```diff
@@ -181,10 +182,7 @@ class PlantProjectSnippet extends React.Component {
 
               <View style={styles.costContainer}>
                 <Text style={styles.costText}>
-                  <NumberFormat
-                    currency={currency}
-                    data={specsProps.treeCost}
-                  />
+                  {formatNumber(specsProps.treeCost, null, currency)}
```
Comment (norbertschuler) [inline app/components/PlantProjects/PlantProjectSnippet.native.js]: Why did you replace the `NumberFormat` component with a direct call to `formatNumber` here? I just want to understand when to use each of them correctly so I do it correctly too.

Comment (mahmed0715) [reply app/components/PlantProjects/PlantProjectSnippet.native.js]: Actually you can see in the NumberFormat component we call the util method formatNumber, so it doesnt matter if you use the component or utils. @norbertschuler 

Feedback requested

Comment (norbertschuler) [reply app/components/PlantProjects/PlantProjectSnippet.native.js]: Thank you. So I think we should use the component if suitable and not use the method `formatNumber` directly.

**Comment by norbertschuler** [comment]
thank you, I will merge this into my branch

**Pull request closed without merging**